### PR TITLE
feat: Add driving license skin in strapi

### LINKED ISSUE #441: Create driving license skin in didComm flow

### PR DESCRIPTION
**Pull request by talwinder50**
closes #441

Signed-off-by: talwinder.kaur <talwinder.kaur@securekey.com>

### CODE REVIEW

File: test/bdd/fixtures/scripts/strapi_configure.sh [hunk 1]
```diff
@@ -52,6 +52,11 @@ GENERATE_CCS_COMMAND="strapi generate:api creditcardstatements UserID:string VcM
 
 $GENERATE_CCS_COMMAND
 
+# generate the drivinglicenses and model
```
Comment (DRK3) [inline test/bdd/fixtures/scripts/strapi_configure.sh:55]: Not sure if you meant for driving license to be one word here (but maybe you did based on the variable name)

File: test/bdd/fixtures/scripts/strapi_configure.sh [hunk 2]
```diff
@@ -176,6 +181,17 @@ if [ "$result" != "null" ]
         echo "error insert creditcardstatements data in strapi: $result"
 fi
 
+# Add driving license data for above created user
+result=$(curl --header "Content-Type: application/json" --header "Authorization: Bearer $token" \
+   --request POST \
+   --data '{"userid":"100","vcmetadata":{"@context":["https://www.w3.org/2018/credentials/v1","https://trustbloc.github.io/context/vc/examples-ext-v1.jsonld"],"name":"Driving License","description":"Driving License of Mr.John Smith"},"stmt":{"description":"Driving License 2019-2024","number":"xxxxx-xxxxx-26206","customer":{"@type":"Person","name":"Jane Doe"},"expiryDate":"2024-06-30T12:00:00", "sex": "Male", "issueyDate":"2019-06-30T12:00:00", "address" : "29 Tan Street", "height" : "159 cm", "class": "g"}}' \
```
Comment (DRK3) [inline test/bdd/fixtures/scripts/strapi_configure.sh:187]: Space after "Mr."

Comment (DRK3) [reply test/bdd/fixtures/scripts/strapi_configure.sh:187]: Also typically it's called a "Driver's license" - e.g. Drivers license for Mr. John Smith or Mr. John Smith's driver's license

Comment (DRK3) [reply test/bdd/fixtures/scripts/strapi_configure.sh:187]: Well the driver's vs driving thing would depend on the context - but the "driving license of Mr. John Smith" text sounds odd to me

Comment (talwinder50) [reply test/bdd/fixtures/scripts/strapi_configure.sh]: done

**Pull request merged**